app/assets/stylesheets/spree/frontend/components/checkout/_payment_controls.scss

```diff
@@ -0,0 +1,5 @@
+.payment-controls {
```

Comment:
What about calling this `.payment-method-controls` to reflect what we are really selecting?